provisioning/standalone/docs/example.md

```diff
@@ -0,0 +1,94 @@
+# Example
+
+In this example, we are going to learn how to deploy a simple echo server using **golang**, which will answer to a **GET HTTP request** with a predefined message.
+
+### Create the server
+
+Create a **main.go** file which will contain the source code of the server and **copy/paste** the following code inside **main.go**
```

Comment:
Should add a brief description about what the code does.

Reply:
I commented the code, do you think could be enough?

Reply:
I would personally write something like this:

```suggestion
Create a **main.go** file with the following content to implement a simple HTTP server:
```

As for the comments, you should leave a space after `//`